tests/x11/xterm.pm

```diff
@@ -11,20 +11,14 @@
 use base "x11test";
 use strict;
 use testapi;
-use utils;
 
 sub run() {
-    my $self = shift;
     mouse_hide(1);
     x11_start_program("xterm");
-    sleep 2;
-    type_string "cd\n";
```

Comment:
I don't know. I didn't do this. But it helps to keep the prompt short so I would recommend to keep it in all cases … and delete the comment stating the obvious and also in a wrong way